themes/index.js

```diff
@@ -355,12 +355,17 @@ const themes = {
     bg_color: "09131B",
     border_color: "0c1a25",
   },
-  "rose_pine":{
+  "rose_pine": {
```

Comment:
Although this is a correct change, please don't change the existing themes so that we can test your PR.